proto/core/Tron.proto

```diff
@@ -135,7 +135,7 @@ message Transaction {
     int64 fee_limit = 18;
   }
 
-  raw raw_data = 1;
+  raw data_raw = 1;
```

Comment:
Change the name of this file to avoid double definitions in Tron.pb.h.

@ClarkChenc will this have any unintended side effect ?

Reply:
Double definition of field `protocol_Transaction_raw_data_tag`

Reply:
Yes, it is. `raw_data` is a member of `Transacation`'s member data, and instances of `Transaction' use it frequently.

Reply:
Ok great thanks for the tip. The `data` field in `raw`  is not used ?

Reply:
yes, it is not used currently.

Reply:
Ok thanks @ClarkChenc . Can you check everything and give me the approval once the CI passes ?

Reply:
Everything green in the CI : )